modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -136,6 +136,10 @@
 ;;; |                                               SQL Driver Methods                                               |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+;; TODO -- all this [[temporal-type]] stuff below can be replaced with the more generalized
```

Comment:
Could put in refactoring backlog in the internal company notion, it seems to me that the items there will probably be more likely to get official time allocated to refactoring

Reply:
this also applies to the other umpteen TODOs in this PR, btw.

the basic problem is that the list of stuff you get upon grepping `TODO:` in the source code is basically only a little bit correlated to that table in the internal company notion